trustd/build.rs

```diff
@@ -34,7 +34,9 @@ fn main() {
 }
 
 fn clone_ui() -> io::Result<ExitStatus> {
-    Command::new(GIT_CMD).args(["submodule", "update"]).status()
+    Command::new(GIT_CMD)
+        .args(["submodule", "update", "--init", "--recursive"])
```

Comment:
Ah! I figured this out on my own, not realizing you were attempting to prevent me from having to do that.

Reply:
yep, the idea is not requiring the user to execute additional commands

Reply:
This change fixed my issue, thanks!